presto-hive/src/main/java/io/prestosql/plugin/hive/s3/PrestoS3FileSystem.java

```diff
@@ -764,7 +765,8 @@ private AWSCredentialsProvider createAwsCredentialsProvider(URI uri, Configurati
             return getCustomAWSCredentialsProvider(uri, conf, providerClass);
         }
 
-        throw new RuntimeException("S3 credentials not configured");
+        // No credentials configured. Fall back to the default credentials provider.
```

Comment:
In https://github.com/prestosql/presto/pull/741/files#diff-2902dc1ae71e318999f5a777a0e0176fR155 you didn't find the comment necessary. I think it's not necessary here as well.

Reply:
Removing